setup-container.sh

```diff
@@ -331,6 +339,36 @@ function parse_arguments() {
     fi
 }
 
+function start_existing_container() {
+    if [ -z ${IMAGE_ID} ]
+    then
+        echo "Missing required argument -i IMAGE_ID to start the container"
+        exit 1
+    fi
+    container_id=`docker container ls --all --filter=ancestor=${IMAGE_ID} --format '{{.ID}}'`
+    count=`echo $container_id | wc -w`
+    if [ $count -eq 0 ]
+    then
+        echo "No containers found for image ${IMAGE_ID}"
+        exit 0
+    fi
+    if [ $count -eq 1 ]
+    then
+        container_name=`docker inspect $container_id --format '{{.Name}}'`
+        cname=""
+        if [[ $container_name == /* ]]
+        then
+            CONTAINER_NAME="${container_name:1}"
+        fi
+        start_local_container "existing"
```

Comment:
Just wondering, why we don't use a return value such like 2,3,4 here?

Reply:
@yutongzhang-microsoft can you please clarify a bit more? Did you mean pass in numeric constants instead of the string "existing"?

Reply:
@yutongzhang-microsoft I have updated the code to use numeric constants instead.